Makefile

```diff
@@ -0,0 +1,21 @@
+.DEFAULT_GOAL := help
+SHELL = bash
+
+
+.PHONY: help
+help: ## show this help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: install
+install: ## install dependencies
+	poetry install --with dev
+	poetry run pre-commit install
```

Comment:
pre-commit shouldn't be installed inside poetry, so that's rather `pre-commit install` without `poetry run`